web/app/view/AttributesAliasesController.js

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright 2016 Anton Tananaev (anton.tananaev@gmail.com)
+ * Copyright 2016 Andrey Kunitsyn (abyss@fox5.ru)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+Ext.define('Traccar.view.AttributesAliasesController', {
+    extend: 'Ext.app.ViewController',
+    alias: 'controller.attributesAliases',
+
+    requires: [
+        'Traccar.view.AttributeAliasDialog',
+        'Traccar.model.AttributeAlias'
+    ],
+
+    init: function () {
+        var admin = Traccar.app.getUser().get('admin');
+        this.lookupReference('deviceField').setStore(admin ? 'AllDevices' : 'Devices');
+        this.lookupReference('toolbarAddButton').setDisabled(true);
+        this.lookupReference('toolbarEditButton').setDisabled(true);
+        this.lookupReference('toolbarRemoveButton').setDisabled(true);
+        this.getView().getStore().loadData([], false);
```

Comment:
Is this a way to clear the store? There must be a better more clear way (maybe `removeAll`?).